docs/migration-guides/1.19-1.20.md

```diff
@@ -0,0 +1,82 @@
+# Migrate from 1.19 to 1.20
+
+## Fluent Bit
+
+Kyma 1.20 comes with a redesigned configuration for Fluent Bit. It is redesigned to be up to date with the upstream chart and also be consistent with the rest of the configuration values. The latest configurations can be found [in the chart](https://github.com/kyma-project/kyma/blob/master/resources/logging/charts/fluent-bit/values.yaml#L134). If you have been using overrides for Fluent Bit, compare them with the latest configurations and adapt if necessary. It also now supports many more configuration options for the plugins coming out of the box. For example, you can set the `alias` field for all the input and output plugins to be used as the parent name for the metric. You can read more about it [here](https://docs.fluentbit.io/manual/administration/monitoring#configuring-aliases).  
```

Comment:
Although it's not "wrong" or anything, if possible, it's nice to make the links more descriptive and avoid links like `here`, `this`,  as mentioned in https://kyma-project.io/community/guidelines/content/#links-links 🙂 

```suggestion
Kyma 1.20 comes with a redesigned configuration for Fluent Bit. It is redesigned to be up to date with the upstream chart and also be consistent with the rest of the configuration values. The latest configurations can be found in the [chart](https://github.com/kyma-project/kyma/blob/master/resources/logging/charts/fluent-bit/values.yaml#L134). If you have been using overrides for Fluent Bit, compare them with the latest configurations and adapt if necessary. It also now supports many more configuration options for the plugins coming out of the box. For example, you can set the `alias` field for all the input and output plugins to be used as the parent name for the metric. You can read more about it in this [Fluent Bit manual](https://docs.fluentbit.io/manual/administration/monitoring#configuring-aliases).  
```